build/build-image/cross/Dockerfile

```diff
@@ -15,16 +15,22 @@
 # This file creates a standard build environment for building cross
 # platform go binary for the architecture kubernetes cares about.
 
-FROM  golang:1.4.2
+FROM  golang:1.5.3
```

Comment:
No, I've bumped it just so this PR builds. 
As I said above, I will rebase this one after something like #14873 is merged.